### DISCUSSION TITLE
Uncertainty Tag?

### DISCUSSION THREAD

**Discussion by MelvinWilde** [DHBern/BeNASch#6] [Suggestions]
There should be a "Uncertainty"-Tag denominating, when it is unclear whether a reference/ word is a person or a location or an organisation or many more for example. This could really help visualize and conceptualize uncertainties in medieval and ancient texts and other types of documents.

**Comment by raykyn** [comment]
Hi Melvin, sorry for the wait for an answer!
I think what you're requesting is already covered. Currently, you can write UNK (for "unknown") instead of a specific entity categorization. Although I just checked the schema document and this seems to be missing there, I'll make this an issue to add the info. 

Does this cover your use case?
